src/style.css

```
@@ -1 +1,58 @@
 /* add styles here */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
...  {
```
Consider using commas to group selectors